src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -214,8 +222,16 @@ const ProjectSelector = createReactClass({
 
     return (
       <MenuItem {...menuItemProps}>
-        {project.isBookmarked && <span className="icon-star-solid bookmark " />}
-        {label}
+        {/*ideally an icon here...*/}
+        <ProjectBadge>
+          {!platforms || !platforms.length ? (
+            <EmptyPlatformicon />
```

Comment:
This logic would be better inside of `PlatformList` so that we can re-use it. I would have a prop like `showEmpty` to determine visibility just in case other components don't want the placeholder.